### PR TITLE
feat(blog): tenant or namespaces, pick at least one

### PR DESCRIPTION
**Pull request by prometherion**
screencapture-localhost-8000-blog-multi-tenancy-options-for-kubernetes-tenant-or-namespaces-pick-at-least-one-html-2021-12-31-11_19_02

### CODE REVIEW

File: docs/blog/index.html [hunk 1]
```diff
@@ -128,6 +128,27 @@ <h1 class="landing-title">Clastix blog</h1>
...
+              <a href="/blog/multi-tenancy-options-for-kubernetes-tenant-or-namespaces-pick-at-least-one.html">
+                Multi-tenancy options for Kubernetes:<br/>
+                Tenant, or Namespaces…? Pick at least one!
+              </a>
+            </h1>
+          </div>
+          <div class="row">
+            <div class="col-md-12">
+              <p class="section-blog">
+                With the accelerating adoption of <strong>Kubernetes</strong>strong> the dilemma of multi-tenancy is gaining more traction and being more hot and relevant. Implementing a multi-tenancy architecture is proving to be one of the more strategic decisions that need to be considered when implementing Kubernetes-based infrastructure, as it has long term implications to operational efficiency, scale velocity and costs.
```
Comment (bsctl) [inline docs/blog/index.html:142]: <strong>Kubernetes</strong>

File: docs/blog/index.html [hunk 2]
```diff
@@ -128,6 +128,27 @@ <h1 class="landing-title">Clastix blog</h1>
   ============================-->
     <section id="master">
         <div class="container">
+          <div class="page-header">
+            <h1>
+              <a href="/blog/multi-tenancy-options-for-kubernetes-tenant-or-namespaces-pick-at-least-one.html">
+                Multi-tenancy options for Kubernetes:<br/>
+                Tenant, or Namespaces…? Pick at least one!
```
Comment (bsctl) [inline docs/blog/index.html:135]: please, remove dots ... from the title

`Multi-tenancy options for Kubernetes: Tenant, or Namespaces? Pick at least one!`

**Review by bsctl — CHANGES_REQUESTED** [review]
LGTM, pls just resolve the comments and then merge.

File: docs/blog/index.html (hunk 1 above)
Comment (prometherion) [reply docs/blog/index.html:142]: I don't get this comment, my bad.

Comment (bsctl) [reply docs/blog/index.html:142]: There is a strong one more time <strong>Kubernetes</strong>strong>

**Review by bsctl — APPROVED** [review]

**Pull request merged**